src/main/java/com/jcabi/xml/XMLDocument.java

```diff
@@ -155,7 +155,7 @@
      * @param text XML document body
      */
     public XMLDocument(@NotNull(message = "XML text can't be NULL")
-        final String text) {
+                       final String text) {
```

Comment:
@Suseika we don't indent arguments that far, please make it just 4 spaces as it was before, here and in the rest of the places you have changed